### ISSUE TITLE
Default :info flash is not an Info flash, it is a Success flash

### ISSUE THREAD

**Issue by njwest** [phoenixframework/phoenix#5795]
tl;dr `:info` flashes are not really `:info`, they're `:success`. 

When I `put_flash(conn, :info, message)`, I don't expect the flash will have green success classes and a "Success!" header.

## Example
This code:

![Screenshot 2024-04-28 at 3 59 14 PM](https://github.com/phoenixframework/phoenix/assets/3742496/25d4c953-016b-46d5-84da-4203e8d27be6)

Results in this flash:

![Screenshot 2024-04-28 at 4 01 34 PM](https://github.com/phoenixframework/phoenix/assets/3742496/3211c9f4-d538-46a7-a1dc-08661fd087c0)

Whether we want to say the `Reset Password` operation not erroring out is a success is a matter of professional opinion, but from a pure code perspective, my expectation is `put_flash(:info, message)` would pop a generic informational flash to the user and `:success` would pop the "Success!" toast.

## Potential improvement
In my own projects I've wired `@kind == :success` to the "Success!" header and `:info` to a more vanilla "Info" flash styling/header in the `flash/1` core component. 

I can attempt to open a PR to add this to the core components and to the generators if desirable. I realize this is a departure from how these flashes have been by default for some time, but IMHO it'll reduce confusion.

**Comment by SteffenDE** [comment]
We decided to change the color to blue and remove the title by default https://github.com/phoenixframework/phoenix/pull/6065.
